lisa/sut_orchestrator/libvirt/transformers.py

```diff
@@ -571,7 +570,11 @@ def _install_libvirt(runbook: schema.TypedSchema, node: Node, log: Logger) -> No
         libvirt_version = libvirt_installer._get_version()
         log.info(f"Already installed! libvirt version: {libvirt_version}")
         _fix_mariner_installation(node=node)
-        node.reboot(time_out=900)
+
+    node.execute("systemctl enable libvirtd", shell=True, sudo=True)
+    node.execute("systemctl enable virtnetworkd", shell=True, sudo=True)
+    log.info("Enabled libvirtd and virtnetworkd services")
```

Comment:
Do these steps apply to the older 6.X Libvirt as well? If not, we should execute them conditionally based on the the version.

Reply:
by 6.x do you mean by the Libvirt in `ch` branch? I am not sure how this will work when compiled within our test pipelines.

If installed with rpms, `virtnetworkd.service` will be a service packaged in our libvirt rpms.